config/app_demo/app_raft_1.ini

```diff
@@ -15,15 +12,17 @@ wal.dir = ./node_1/rocksdb
 [netadmin]
 ip.port = 0.0.0.0:50065
 
-[store]
+[cluster]
 persistence.type = raft
+cluster.conf = 1#1@0.0.0.0:5253|50055|55055|65055|75055|85055,2@0.0.0.0:5254|50056|55056|65055|75055|85055,3@0.0.0.0:5255|50057|55057|65055|75055|85055
```

Comment:
If the three nodes are running on the same host, is there port conflict on 65055|75055|85055?

Reply:
good spot. updated.